generate-stackbrew-library.sh

```diff
@@ -3,12 +3,12 @@ set -Eeuo pipefail
 
 declare -A aliases=(
 	[2.6]='2 latest'
-	[2.7-rc]='rc'
+	[2.7]='rc'
```

Comment:
Shouldn't this be like following and remove line above (`[2.6]='2 latest'`)?

```suggestion
	[2.7]='2 latest'
```